src/init.cpp

```diff
@@ -1408,8 +1409,21 @@ bool AppInitMain(boost::thread_group& threadGroup, CScheduler& scheduler)
             threadGroup.create_thread(&ThreadScriptCheck);
     }
 
-    if (!sporkManager.SetSporkAddress(GetArg("-sporkaddr", Params().SporkAddress())))
-        return InitError(_("Invalid spork address specified with -sporkaddr"));
+    std::vector<std::string> vSporkAddresses;
+    if(mapMultiArgs.count("-sporkaddr")) {
+        vSporkAddresses = mapMultiArgs.at("-sporkaddr");
+    } else {
+        vSporkAddresses = Params().SporkAddresses();
+    }
+    for(const auto& address: vSporkAddresses) {
+        if (!sporkManager.SetSporkAddress(address))
+            return InitError(_("Invalid spork address specified with -sporkaddr"));
```

Comment:
bodies of an if should either be on the same line or in brackets 
```
If an if only has a single-statement then-clause, it can appear on the same line as the if, without braces. In every other case, braces are required, and the then and else clauses must appear correctly indented on a new line.
```